beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -1082,6 +1088,42 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
             }
         }
     }
+
+    /// Creates a string visualization of the current state of the chain, to make it easier for debugging and understanding
+    /// where sync is up to from glancing at the logs.
+    ///
+    /// This produces a string of the form: [⏳⏬⏬⏬⏬]
+    /// to indicate the current buffer state of the chain. The symbols are defined on each of the
+    /// batch states. See [BatchState::visualize] for symbol definitions.
+    fn visualize_batch_state(&self) -> String {
+        let mut visualization_string = String::with_capacity((BATCH_BUFFER_SIZE * 3) as usize);
+
+        // Start of the block
+        visualization_string.push('[');
+
+        for mut batch_index in 0..BATCH_BUFFER_SIZE {
+            if let Some(batch) = self
+                .batches
+                .get(&(self.processing_target + batch_index as u64 * EPOCHS_PER_BATCH))
+            {
+                visualization_string.push_str(batch.visualize());
+                if batch_index != BATCH_BUFFER_SIZE {
+                    // Add a space in between elements
+                    visualization_string.push(' ');
+                }
+            } else {
+                // No batch exists, it is on our list to be downloaded
+                // Fill in the rest of the gaps
+                while batch_index < BATCH_BUFFER_SIZE {
+                    visualization_string.push('');
```

Comment:
This doesn't render for me, but I think it's the character you want lol (the math symbol that never renders)